doc/website/index.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: splash
+permalink: /
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/mm-home-page-feature.jpg
+  actions:
+    - label: "<i class='fas fa-download'></i> Install now"
+      url: "/get-started/setup/"
+excerpt: >
+  Develop and design Hardware seamlessly with the power of ROHD.<br />
+  <small><a href="https://github.com/intel/rohd/releases">Latest release v0.4.1</a></small>
```

Comment:
Can we add this as a check in the version check test so that we don't release with the wrong version number?

Reply:
Added the test.